src/components/BitcoinAmountInput.tsx

```diff
@@ -77,7 +149,6 @@ const BitcoinAmountInput = forwardRef(
             data-display-value={field.value?.displayValue}
             name={field.name}
             autoComplete="off"
-            type={inputType.type}
```

Comment:
Can you let me know the reason for removing the `type` attribute?
e.g. this disabled up/down arrows for number inputs.

Reply:
I removed the `type` attribute due to inconsistencies in how different browsers handle it. For example, in Brave and Chrome, only numbers are allowed, while Firefox permits strings as well. Additionally, changing the HTML lang attribute to "IT" allows the use of ',' as a separator for Italian number formats.

Removing the `type` attribute helps ensure a more consistent input experience across different browsers.